google/cloud/sql/connector/instance_connection_manager.py

```diff
@@ -52,9 +54,16 @@
 APPLICATION_NAME = "cloud-sql-python-connector"
 SERVER_PROXY_PORT = 3307
 
-# The default delay is set to 55 minutes since each ephemeral certificate is only
-# valid for an hour. This gives five minutes of buffer time.
-_delay: int = 55 * 60
+# default_refresh_buffer is the minimum amount of time for which a
+# certificate must be valid to ensure the next refresh attempt has adequate
+# time to complete.
+_default_refresh_buffer: int = 5 * 60
+
+# iamAuthRefreshBuffer is the minimum amount of time for which a
+# certificate holding an Access Token must be valid. Because some token
+# sources are refreshed with only ~60 seconds before expiration, this value
+# must be smaller than the defaultRefreshBuffer.
```

Comment:
```suggestion
# _iam_auth_refresh_buffer is the amount of time before a refresh's result expires
# that a new refresh operation begins when IAM DB AuthN is enabled. Because token 
# sources may be cached until ~60 seconds before expiration, this value must be smaller
# than default_refresh_buffer.
```